hospitalFactory.java

```diff
@@ -0,0 +1,13 @@
+
+public class hospitalFactory {
+	
+	public hospital getHospital(String quirophan) {
+		if("yes".equals(quirophan)) {

```

Comment:
The factory returns Low / High Complexity Hospitals. Instead of using `quirophan: yes / no` it would be clearer to name the argument `complexity` or `type` and use `lowComplexity / highComplexity` as values.